ci/concourse/image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang
```

Comment:
Should we base this on golang alpine or is it too much of a PITA? I think our concourse server has struggled in the past trying to run too many heavy images.

Reply:
Are we going to need golang either way?

Reply:
golang provides the go binary. I'll test with golang-alpine

Reply:
our integration tests require gcc, so it wasn't worth the switch

Reply:
our tests require gcc, so this wasn't worth it